tests/npm-tsoa-app/tsoa-support/routes.ts.tmpl

```diff
@@ -0,0 +1,191 @@
+// A tsoa routes template for CCF. Needs to be post-processed by postprocess.js.
```

Comment:
This comment seems to have been added manually but the rest of the file seems to have been generated by `tsoa`?

Reply:
No, this whole file is a template that gets processed by tsoa. It's not generated.